### ISSUE TITLE
Pretraining

### ISSUE THREAD

**Issue by navidivan** [Transconnectome/SwiFT#3]
I am abit confused, (maybe comes from unfamiliarity)
main.py does not parse pretraining, use_contrastive, and alot of other ARGS. They are used later but they are not parsed.

Running the colab demo, gives the following error for the pretrained model:

TRAINER_ARGS='--accelerator gpu --max_epochs 10 --precision 16 --num_nodes 1 --devices 1' # specify the number of gpus as '--devices'
MAIN_ARGS='--loggername tensorboard --classifier_module v6 --dataset_name S1200 --image_path /content/drive/MyDrive/SwiFT_tutorial_official/SwiFT_tutorial_official/MNI_to_TRs'
DATA_ARGS='--batch_size 2 --num_workers 2  --input_type rest'
DEFAULT_ARGS='' #--project_name '
OPTIONAL_ARGS='--c_multiplier 2 --last_layer_full_MSA True \
 --clf_head_version v1 --downstream_task sex \ --pretraining --use_contrastive --contrastive_type 1'

!python ./SwiFT/project/main.py $TRAINER_ARGS $MAIN_ARGS $DEFAULT_ARGS $DATA_ARGS $OPTIONAL_ARGS $RESUME_ARGS \
--dataset_split_num 1 --seed 1 --learning_rate 5e-5 --model swin4d_ver7 --depth 2 2 6 2 --embed_dim 36 --sequence_length 20 --first_window_size 4 4 4 4 --window_size 4 4 4 4 --img_size 96 96 96 20

error:
main.py: error: unrecognized arguments:  --pretraining

The main.py does not parse alot of the inputs.

**Comment by navidivan** [comment]
this was a syntax error by me. could make it work.
